sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -11,12 +11,14 @@ import {
   APPLICATION_NUMBER_LENGTH,
   DisbursementScheduleStatus,
 } from "@sims/sims-db";
+import { JsonMaxSize } from "../../../utilities/class-validation";
 
 export class SaveApplicationAPIInDTO {
   /**
    * Application dynamic data.
    */
   @IsObject()
+  @JsonMaxSize(20480)
```

Comment:
We try to avoid using the numbers directly and the recommended way would be to create a const.
It could be something trivial as `PAYLOAD_2MB` or something similar.